pwndbg/next.py

```diff
@@ -103,6 +108,18 @@ def break_next_call(symbol_regex=None):
         if ins.symbol and re.match('%s$' % symbol_regex, ins.symbol):
             return ins
 
+
+def break_next_ret(address=None):
+    while pwndbg.proc.alive:
+        ins = break_next_branch(address)
+
+        if not ins:
+            break
+
+        if capstone.CS_GRP_RET in ins.groups:
```

Comment:
Same for `nextsc` - it at least breaks instead of loop forever:
```
pwndbg> nextsc
Traceback (most recent call last):
  File "/home/dc/installed/pwndbg/pwndbg/commands/__init__.py", line 99, in __call__
    return self.function(*args, **kwargs)
  File "/home/dc/installed/pwndbg/pwndbg/commands/__init__.py", line 191, in _OnlyWhenRunning
    return function(*a, **kw)
  File "/home/dc/installed/pwndbg/pwndbg/commands/next.py", line 65, in next_syscall
    while pwndbg.proc.alive and not pwndbg.next.break_next_interrupt() and pwndbg.next.break_next_branch():
  File "/home/dc/installed/pwndbg/pwndbg/next.py", line 80, in break_next_interrupt
    ins = next_int(address)
  File "/home/dc/installed/pwndbg/pwndbg/next.py", line 38, in next_int
    ins = pwndbg.disasm.one(pwndbg.regs.pc)
  File "/home/dc/installed/pwndbg/pwndbg/disasm/__init__.py", line 108, in one
    for insn in get(address, 1):
  File "/home/dc/installed/pwndbg/pwndbg/disasm/__init__.py", line 128, in get
    i = get_one_instruction(address)
  File "/home/dc/installed/pwndbg/pwndbg/memoize.py", line 48, in __call__
    value = self.func(*args, **kwargs)
  File "/home/dc/installed/pwndbg/pwndbg/disasm/__init__.py", line 100, in get_one_instruction
    pwndbg.disasm.arch.DisassemblyAssistant.enhance(ins)
  File "/home/dc/installed/pwndbg/pwndbg/disasm/arch.py", line 57, in enhance
    print(enhancer.dump(instruction))
  File "/home/dc/installed/pwndbg/pwndbg/disasm/arch.py", line 254, in dump
    rv.append('       access   = %s' % (access.get(op.access, op.access)))
AttributeError: 'ArmOp' object has no attribute 'access'
```